stubs/tqdm/tqdm/contrib/discord.pyi

```diff
@@ -79,8 +81,9 @@ class tqdm_discord(tqdm_auto[_T]):
         gui: bool = ...,
         **kwargs,
     ) -> None: ...
+    # type: ignore[override]
     def display(
-        self, msg: str | None = ..., pos: int | None = ..., close: bool = ..., bar_style=..., check_delay: bool = ...
+        self, *, msg: str | None = ..., pos: int | None = ..., close: bool = ..., bar_style=..., check_delay: bool = ...
```

Comment:
mypy failure is due to these parameters being keyword-only here but positional-or-keyword in the parent:

https://github.com/python/typeshed/blob/90842a4520785e0f6ad5d00fccac6a57ca6c6ad3/stubs/tqdm/tqdm/std.pyi#L224

Reply:
This seems like an issue in the source library, the implementation for this function is keyword-only: https://github.com/tqdm/tqdm/blob/master/tqdm/contrib/discord.py#L126

Reply:
Then we'll need to add a type ignore here with a comment.